validation.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs')
+const path = require('path')
+const yaml = require('js-yaml')
+const Ajv = require('ajv')
+
+const yamlFilePath = path.join(
+  __dirname,
+  '/fixtures/github/make_dispatches.yaml'
+)
+const yamlContent = fs.readFileSync(yamlFilePath, 'utf8')
+const yamlData = yaml.load(yamlContent)
+
+const schemaFilePath = path.join(__dirname, '/schema/jsonschema.json')
+const schema = JSON.parse(fs.readFileSync(schemaFilePath, 'utf8'))
+
+const ajv = new Ajv()
+const validate = ajv.compile(schema)
+
+const valid = validate(yamlData)
+
+if (valid) {
+  console.log('Successful validation')
+} else {
+  console.error('Validation error:', validate.errors)
+  process.exit(1)
+}
```

Comment:
This is a test file, you don't need to do a process exit!

We are using Jest to create the tests cases: https://jestjs.io/es-ES/

You must define a tests case and use `expect` to validate the expected results, and the tests will pass or fail according to the results of the expected evaluation. 

See for example: https://github.com/prefapp/action-make-state-repos-dispatches/blob/f7a81d0ed6282a95f0c83f485789050aa701a057/__tests__/dispatcher.test.js#L75-L84